gnocchi/tests/test_indexer.py

```diff
@@ -1121,9 +1124,6 @@ def test_get_metric_with_bad_uuid(self):
         self.assertEqual([], self.index.list_metrics(
             attribute_filter={"=": {"id": e1}}))
 
-    def test_get_metric_empty_list_uuids(self):
```

Comment:
what happens in this scenario now?

Reply:
Same things. it's basically `{"in" : {"id": []}}`

it's not a special case anymore that's why I removed the test

Reply:
does it crash? i'm pretty sure it was there to make sure it didn't throw some SQL error which it did previously with empty list